src/governance-classes.cpp

```diff
@@ -512,9 +509,31 @@ CSuperblock(uint256& nHash)
     DBG( cout << "CSuperblock Constructor End" << endl; );
 }
 
-void 
-CSuperblock::
-ParsePaymentSchedule(std::string& strPaymentAddresses, std::string& strPaymentAmounts)
+bool CSuperblock::IsValidBlockHeight(int nBlockHeight)
+{
+    // SUPERBLOCKS CAN HAPPEN ONLY after hardfork and only ONCE PER CYCLE
+    return nBlockHeight >= Params().GetConsensus().nSuperblockStartBlock &&
+            ((nBlockHeight % Params().GetConsensus().nSuperblockCycle) == 0);
+}
+
+CAmount CSuperblock::GetPaymentsLimit(int nBlockHeight)
+{
+    const Consensus::Params& consensusParams = Params().GetConsensus();
+
+    if(!IsValidBlockHeight(nBlockHeight))
+        return 0;
```

Comment:
Since we all seem to agree that it's better to keep the braces, can we at least do that for new code ?